File: app/src/main/java/nl/rsdt/japp/jotial/data/structures/area348/DeletedInfo.java
package nl.rsdt.japp.jotial.data.structures.area348;

import android.os.Parcel;
import android.os.Parcelable;

public class DeletedInfo implements Parcelable {
    public boolean verwijderd;
    protected DeletedInfo(Parcel in) {
        verwijderd = true;
    }
    @Override
    public int describeContents() {
        return 0;
    }

    @Override
    public void writeToParcel(Parcel dest, int fags) {

    }
    public static final Creator<DeletedInfo> CREATOR = new Creator<DeletedInfo>() {
        @Override
        public DeletedInfo createFromParcel(Parcel in) {
            return new DeletedInfo(in);
        }

        @Override
        public DeletedInfo[] newArray(int size) {
            return new DeletedInfo[size];
        }
    };
}

File: app/src/main/java/nl/rsdt/japp/jotial/data/structures/area348/DeletedInfo.kt
package nl.rsdt.japp.jotial.data.structures.area348

import android.os.Parcel
import android.os.Parcelable

class DeletedInfo protected constructor(`in`: Parcel) : Parcelable {
    var verwijderd: Boolean = false

    init {
        verwijderd = true
    }

    override fun describeContents(): Int {
        return 0
    }

    override fun writeToParcel(dest: Parcel, fags: Int) {

    }

    companion object {
        val CREATOR: Parcelable.Creator<DeletedInfo> = object : Parcelable.Creator<DeletedInfo> {
            override fun createFromParcel(`in`: Parcel): DeletedInfo {
                return DeletedInfo(`in`)
            }

            override fun newArray(size: Int): Array<DeletedInfo> {
                return arrayOfNulls(size)
            }
        }
    }
}
